# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/XRWebGLLayer.cpp
# Commit: dd00e2da3a0f
# Full Hash: dd00e2da3a0f11a7cc0af9e5ece21a339f4393eb
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 69cc1de6f754 -r dd00e2da3a0f dom/canvas/XRWebGLLayer.cpp
--- a/dom/canvas/XRWebGLLayer.cpp	Wed Jun 10 21:57:19 2020 +0000
+++ b/dom/canvas/XRWebGLLayer.cpp	Wed Jun 10 22:21:02 2020 +0000
@@ -261,7 +261,6 @@
 
 void XRWebGLLayer::SessionEnded() {
   DeleteFramebuffer();
-  mWebGL->ClearVRFrame();
 }
 
 }  // namespace dom